CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release history
 
+## Version 0.5
+
+Catchy phrase here :)
```

Comment:
Ideas: "High five!" or "[Famous Five](https://en.wikipedia.org/wiki/The_Famous_Five_(novel_series))" or "Gimme five" or "[five by five](https://www.urbandictionary.com/define.php?term=five%20by%20five)" or "[Man alive](https://en.wikipedia.org/wiki/List_of_British_bingo_nicknames)"

Reply:
I like High five! most - for others people might not get a reference :)